style.css

```diff
@@ -1,20 +1,25 @@
 /* globel reset */
```

Comment:
Could suggest moving global rules and utilities into separate CSS file for organization